blog/_posts/2022-02-24-rancher-desktop-container-runtimes.md

```diff
@@ -0,0 +1,118 @@
+---
+slug: "rancher-desktop-container-runtimes"
+date: 2022-02-24
+author: milas
+layout: blog
+title: "Rancher Desktop: Should You Use containerd Or dockerd?"
+image: "/assets/images/rancher-desktop-container-runtimes/title.jpg"
+image_caption: 'Photo by <a rel="noopener noreferrer" href="https://unsplash.com/photos/u0vgcIOQG08">Jens Lelie</a> on Unsplash'
+description: "Configure Tilt to build images with kim for use with Rancher Desktop"
+tags:
+- docker
+- moby
+- containerd
+- rancher
+- kubernetes
+---
+
+[Rancher Desktop][rancher-desktop], a lightweight and local Kubernetes cluster solution, has been evolving _fast_ over the past few months!
+
+Early releases used [containerd][] behind-the-scenes with an experimental CLI tool, [`kim`][kim], to build images.
+I wrote up a blog post about how easy it was to integrate with Tilt: [Writing Yet Another Custom Image Builder][blog-kim-extension].
+
+Since then, however, Rancher Desktop has started also supporting Docker (aka `dockerd (moby)`) as an alternative to containerd.
+Furthermore, [`kim`][kim] is no longer under active development.
+
+While this might seem like a lot of churn, none of this is necessarily a bad thing!
+In fact, when I was first writing about all this, our CTO, Nick, even predicted some of these changes.
+
+If you're currently a Docker Desktop user who's interested in checking out Rancher Desktop, you're also in luck:
+I've updated our [Switch from Docker Desktop to Rancher Desktop in 5 Minutes][blog-rancher-desktop-switch] blog post! 🎉
+
+While that post is a great place to start, let's take a peek at the container runtime options in a bit more detail.
+Why might you choose one over the other?
+How do they work with Tilt?
+
+### containerd
+The default container runtime in Rancher Desktop is [containerd][].
+
+These days, it's also the de facto standard container runtime used in production Kubernetes installs, so it's not surprising it's the default here!
```

Comment:
i would probably link to this post, which i think does a good job of explaining WHY it's the standard
https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/

i think a lot of people get confused about the difference between docker the image builder and docker the container runtime.